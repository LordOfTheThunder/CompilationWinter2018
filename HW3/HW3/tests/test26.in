struct potato {
	int x;
};
void TestStructParami(struct potato t) {
	int s;
}
