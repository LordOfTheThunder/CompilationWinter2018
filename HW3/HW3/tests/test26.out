---end scope---
t potato -1
s INT 0
---end scope---
print (STRING)->VOID 1
printi (INT)->VOID 2
struct potato {INT x}
TestStructParami (potato)->VOID 3
